src/Domain/Kernel.php

```diff
@@ -49,4 +55,20 @@ public static function getRequiredFiles(): array
             // '.gitignore',
         ];
     }
+
+    /**
+     * Returns the list of Insights required on root.
+     *
+     * @return array<string>
+     */
+    public static function getGlobalInsights(): array
```

Comment:
Don't know if it's the best place to put this function.

Reply:
Not sure either, but it's easy to change later on.